src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -537,6 +556,41 @@ export default class FilterManager extends WebGLManager
         {
             this.pool = {};
         }
+        this.managedDisplayObjects.forEach((displayObject) =>
+        {
+            displayObject.off('destroyed', this.onManagedDisplayObjectDestroyed, this);
+        });
+        this.managedDisplayObjects = [];
+    }
+
+    /**
+     * Destroys FilterState of specified target
+     *
+     * @param {PIXI.DisplayObject} target, which FilterState should be destroyed
+     */
+    destroyFilterStateByTarget(target)
```

Comment:
Why create a separate function? Wouldn't it just be easier to inline in the destroy handler?